merkato/utils/database_utils.py

```diff
@@ -73,21 +73,21 @@ def create_exchanges_table():
 		finally:
```

Comment:
The indentation in this file is different from the rest of the project

Reply:
done